### PR TITLE
Cross-sell: Add tests

### PR DESCRIPTION
**Pull request by mlahargou**
This adds some basic tests for our cross sell component. We assert
that we can check and uncheck different products and they are
reflected in the price in the component as well as in the cart
drawer.

QA:
--
These should run in the playwright ci job.

Connects: https://github.com/ifixit/react-commerce/issues/1202

### CODE REVIEW

**Comment by jordycosta** [comment]
QA 🟢 

cross_sell

The **cross_sell.spec.ts** test is being ran [by playwright](https://github.com/iFixit/react-commerce/actions/runs/3851147894/jobs/6562243993) 👍

File: frontend/tests/playwright/product/cross_sell.spec.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      let currentProductPrice = null;
+      let otherProductTitles = [];
+
+      for (let i = 0; i < count; i++) {
+         const product = products.nth(i);
+
+         const isCurrent = await product.locator('span', { hasText: 'Current item' }).isVisible();
+         if (isCurrent) {
+            currentProductPrice = await product.getByTestId('product-price').textContent();
+            await expect(currentProductPrice).toMatch(/^\$[0-9]+(\.[0-9]{2})/);
```
Comment (ardelato) [inline frontend/tests/playwright/product/cross_sell.spec.ts:28]: Hmm, I am wondering if we should add a utility function for nabbing a product's price as a `float`? This is in reference to #1217

**Review by ardelato — APPROVED** [review]
CR ⚡

File: frontend/tests/playwright/product/cross_sell.spec.ts (hunk 1 above)
Comment (mlahargou) [reply frontend/tests/playwright/product/cross_sell.spec.ts:28]: Probably!

Comment (ardelato) [reply frontend/tests/playwright/product/cross_sell.spec.ts:28]: Noted, I'll add it to the issue

**Pull request merged**